### PR TITLE
Implement unlock token multi-sig motion

### PR DESCRIPTION
**Pull request by davecreaser**
Implement unlock token multi-sig motion.

Part of https://github.com/JoinColony/colonyCDapp/pull/2578

### CODE REVIEW

**Review by bassgeta — APPROVED** [review]
Nice one, tested the CDapp counterpart and everything works :sunglasses:

**Review by rdig — APPROVED** [review]
All good here as well, nothing stands out.

Tested this as part of the review of JoinColony/colonyCDapp/pull/2578

File: src/handlers/multiSig/multiSigCreated/multiSigCreated.ts [hunk 1]
```diff
@@ -26,11 +29,13 @@ export const handleMultiSigMotionCreated: EventHandler = async (
...
-    await getStakedExpenditureClient(colonyAddress);
+  const stakedExpenditureClient = await getStakedExpenditureClient(
+    colonyAddress,
+  );
 
-  const stagedExpenditureClient =
-    await getStagedExpenditureClient(colonyAddress);
+  const stagedExpenditureClient = await getStagedExpenditureClient(
+    colonyAddress,
+  );
```
Comment (iamsamgibbs) [inline src/handlers/multiSig/multiSigCreated/multiSigCreated.ts:38]: Looks like these are just on save formatting changes? Maybe we should open an issue to add a .prettierrc file to block-ingestor

**Review by iamsamgibbs — APPROVED** [review]
Code looks good to me! Tested with the CDapp counterpart and all working as expected.

**Pull request merged**